cmd/s3gw-cosi-driver/main.go

```diff
@@ -1,5 +1,6 @@
 /*
 Copyright 2023 SUSE, LLC.
+Copyright 2024 s3gw maintainers.
```

Comment:
maybe use "contributors" instead of "maintainers"? I think that would be more inclusive than just attributing copyright to a handful of people (even if it doesn't really matter much in the end).

Reply:
Sure! Updated all of those as suggested!